app/models/page/renderer.rb

```diff
@@ -181,11 +181,13 @@ def init_responsive_tables(doc)
     doc.css('table.responsive-table').each do |table|
       thead_ths = table.css('thead th')
 
-      table.search('./tbody/tr').each do |tr|
-        tr.search('./td').each_with_index do |td, i|
-          faux_th = "<th aria-hidden class=\"responsive-table__faux-th\">#{thead_ths[i].children}</th>"
-          
-          td.add_previous_sibling(faux_th)
+      unless thead_ths.empty?
```

Comment:
Prevent error and skip prepending _faux `th`_ when no `th` are found in responsive tables.